### PR TITLE
Build Dex image from dockerfile

### PR DESCRIPTION
**Pull request by emosbaugh**
#### What type of PR is this?

kind/chore

#### What this PR does / why we need it:

Just some cleanup

#### Which issue(s) this PR fixes:

Fixes #NONE

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Does this PR require documentation?

NONE

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -2,7 +2,7 @@ include Makefile.build
 CURRENT_USER := $(shell id -u -n)
 MINIO_TAG ?= RELEASE.2021-09-15T04-54-25Z
 POSTGRES_ALPINE_TAG ?= 10.18-alpine
-DEX_TAG ?= v2.28.1
+DEX_TAG ?= v2.30.0
```
Comment (DanStough) [inline Makefile:5]: Isn't this tag suppose to come from the environment file now? This is just the fallaback, correct?

Comment (emosbaugh) [reply Makefile:5]: yes, fallback

Comment (jala-dx) [reply Makefile:5]: @DanStough the tag is exported at the root, this is a fallback like @emosbaugh says.

Comment (emosbaugh) [reply Makefile:5]: i do think it would be good to change that to `latest` to avoid confusion. not donna do it in this pr

**Review by DanStough — APPROVED** [review]

**Pull request merged**